tests/end-to-end/api/04-direct-message.js

```diff
@@ -517,35 +542,66 @@ describe('[Direct Messages]', function() {
 			request.post(api('im.create'))
 				.set(credentials)
 				.send({
-					usernames: ['rocket.cat', otherUser.username].join(','),
+					usernames: [user.username, otherUser.username].join(','),
 					excludeSelf: true,
 				})
 				.expect(200)
 				.expect('Content-Type', 'application/json')
 				.expect((res) => {
 					expect(res.body).to.have.property('success', true);
 					expect(res.body).to.have.property('room').and.to.be.an('object');
-					expect(res.body.room).to.have.property('usernames').and.to.have.members(['rocket.cat', otherUser.username]);
-					roomId = res.body.room._id;
+					expect(res.body.room).to.have.property('usernames').and.to.have.members([user.username, otherUser.username]);
+					roomIds = { ...roomIds, dm: res.body.room._id };
 				})
 				.end(done);
 		});
 
 		it('should create a self-DM', (done) => {
 			request.post(api('im.create'))
-				.set(credentials)
+				.set(userCredentials)
 				.send({
-					username: adminUsername,
+					username: user.username,
 				})
 				.expect(200)
 				.expect('Content-Type', 'application/json')
 				.expect((res) => {
 					expect(res.body).to.have.property('success', true);
 					expect(res.body).to.have.property('room').and.to.be.an('object');
-					expect(res.body.room).to.have.property('usernames').and.to.have.members([adminUsername]);
+					expect(res.body.room).to.have.property('usernames').and.to.have.members([user.username]);
+					roomIds = { ...roomIds, self: res.body.room._id };
 				})
 				.end(done);
 		});
+
+		async function testRoomFNameForUser(credentials, rid, fullName) {
+			return request.get(api('subscriptions.getOne'))
+				.set(credentials)
+				.query({ roomId: rid })
```

Comment:
being very picky here.. but since there is a `credentials` variable on the scope above, would be good to call this different to avoid confusion..

what about changing to `roomId` as well to make things cleaner 😬 

```suggestion
		async function testRoomFNameForUser(testCredentials, roomId, fullName) {
			return request.get(api('subscriptions.getOne'))
				.set(testCredentials)
				.query({ roomId })
```